runtime/j9vm/java11vmi.c

```diff
@@ -1561,17 +1542,10 @@ JVM_SetBootLoaderUnnamedModule(JNIEnv *env, jobject module) {
 		j9object_t modObj = J9_JNI_UNWRAP_REFERENCE(module);
 		J9ClassLoader *systemClassLoader = vm->systemClassLoader;
 		J9Class *instanceClazz = J9OBJECT_CLAZZ(currentThread, modObj);
-		J9Class *moduleClass = NULL;
-		if (J2SE_SHAPE(vm) < J2SE_SHAPE_B165) {
-			moduleClass = vmFuncs->internalFindKnownClass(currentThread,
-				J9VMCONSTANTPOOL_JAVALANGREFLECTMODULE,
-				J9_FINDKNOWNCLASS_FLAG_INITIALIZE);
-		} else {
-			moduleClass = vmFuncs->internalFindKnownClass(currentThread,
-				J9VMCONSTANTPOOL_JAVALANGMODULE,
-				J9_FINDKNOWNCLASS_FLAG_INITIALIZE);
-		}
 		if (!currentThread->currentException) {
+			J9Class *moduleClass = vmFuncs->internalFindKnownClass(currentThread,
+					J9VMCONSTANTPOOL_JAVALANGMODULE,
+					J9_FINDKNOWNCLASS_FLAG_INITIALIZE);
 			if (!isModuleUnnamed(currentThread, modObj)) {
 				vmFuncs->setCurrentExceptionUTF(currentThread, J9VMCONSTANTPOOL_JAVALANGILLEGALARGUMENTEXCEPTION, "named module was supplied");
 			} else if (!isSameOrSuperClassOf(moduleClass, instanceClazz)) {
```

Comment:
Nitpick: `moduleClass` definition can be moved to this code block.

Reply:
Not without introducing a new `{}` block surrounding that if statement.